platform_plugin_communications/tests/test_tasks.py

```diff
@@ -0,0 +1,132 @@
+"""
+Test for tasks.
+"""
+from unittest.mock import Mock, patch
+
+from django.test import TestCase, override_settings
+
+from platform_plugin_communications.tasks import (
+    perform_delegate_email_batches_to_learners,
+    send_bulk_course_email_to_learners,
+)
+
+
+@override_settings(
+    PLATFORM_PLUGIN_COMMUNICATIONS_BULK_EMAIL_API_BACKEND=(
+        "platform_plugin_communications.edxapp_wrapper.backends.bulk_email_api_p_v1_test"
+    ),
+    PLATFORM_PLUGIN_COMMUNICATIONS_INSTRUCTOR_VIEWS_API_BACKEND=(
+        "platform_plugin_communications.edxapp_wrapper.backends.instructor_views_p_v1_test"
+    ),
+    PLATFORM_PLUGIN_COMMUNICATIONS_COURSE_OVERVIEWS_BACKEND=(
+        "platform_plugin_communications.edxapp_wrapper.backends.course_overviews_p_v1_test"
+    ),
+    PLATFORM_PLUGIN_COMMUNICATIONS_UTIL_QUERY_BACKEND=(
+        "platform_plugin_communications.edxapp_wrapper.backends.util_query_p_v1_test"
+    ),
+    PLATFORM_PLUGIN_COMMUNICATIONS_INSTRUCTOR_TASKS_BACKEND=(
+        "platform_plugin_communications.edxapp_wrapper.backends.instructor_tasks_p_v1_test"
+    ),
+    PLATFORM_PLUGIN_COMMUNICATIONS_COURSEWARE_COURSES_BACKEND=(
+        "platform_plugin_communications.edxapp_wrapper.backends.courseware_courses_p_v1_test"
+    ),
+)
```

Comment:
You can add this to the tests.py settings file

Reply:
Those settings are in `test_settings.py` but it's value is not taken correctly